src/vm/dllimportcallback.cpp

```diff
@@ -1416,6 +1416,7 @@ VOID UMThunkMarshInfo::RunTimeInit()
     {
         // caller pop
         m_cbRetPop = 0;
+        m_cbActualArgSize += numRegistersUsed * STACK_ELEM_SIZE;
```

Comment:
The should be updated for both calling conventions - the args stack size is not different between them, the only difference is whether we pop them or not in the UMThunkStub.
So this new line should be moved before the `if`.